src/ActionMonitor/Monitors/PostMonitor.php

```diff
@@ -58,6 +58,10 @@ public function callback_transition_post_status( $new_status, $old_status, WP_Po
 			return;
 		}
 
+        if(defined("REST_REQUEST") && REST_REQUEST) {
```

Comment:
@DerthVedder can you update your code formatting to match the rest of the file?

Reply:
No problem. Updated.